easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.0-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,59 @@
+# by Ward Poelmans <wpoely86@gmail.com>
+
+easyblock = 'ConfigureMake'
+
+name = 'PLUMED'
+version = '2.6.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.plumed.org'
+description = """PLUMED is an open source library for free energy calculations in molecular systems which
+ works together with some of the most popular molecular dynamics engines. Free energy calculations can be
+ performed as a function of many order parameters with a particular  focus on biological problems, using
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = ['https://github.com/plumed/plumed2/releases/download/v%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['8a760a637225eeb97e84b748e992ebc0730f71cdb4737af545445132380bfff9']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('GSL', '2.6'),
+    ('libmatheval', '1.1.11'),
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('Boost', '1.72.0'),
+]
+
+preconfigopts = 'env FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
+configopts = ' --exec-prefix=%(installdir)s --enable-gsl --enable-modules=all --enable-python'
+configopts += ' --enable-boost_graph --enable-boost_serialization '
+configopts += ' --enable-asmjit '
+prebuildopts = 'source sourceme.sh && '
+
+# make sure that ld.gold linker is used
+# requires to work around problems like "ld: BFD (GNU Binutils) 2.30 assertion fail elf.c:3564"
```

Comment:
And "required", should probably also state that this appliles to the intel build, but for simplicity we keep them identical